Formula/git-appraise.rb

```diff
@@ -0,0 +1,24 @@
+require "language/go"
+
+class GitAppraise < Formula
+  desc "Distributed code review system for Git repos"
+  homepage "https://github.com/google/git-appraise"
+  url "https://github.com/google/git-appraise/archive/v0.6.tar.gz"
+  sha256 "5c674ee7f022cbc36c5889053382dde80b8e80f76f6fac0ba0445ed5313a36f1"
+  head "https://github.com/google/git-appraise.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    (buildpath/"src/github.com/google").mkpath
+    ln_s buildpath, buildpath/"src/github.com/google/git-appraise"
+
+    system "go", "build", "-o", bin/"git-appraise", "./src/github.com/google/git-appraise/git-appraise"
+  end
+
+  test do
+    system bin/"git-appraise", "help"
```

Comment:
This needs to be a test that exercises and verifies the functionality of the software.

Reply:
kinda like `git-lfs`, if i do significant testing, i'll need `git`.

should i make `git` a `depends` or `recommends` then? it's not listed under `git-lfs`, which is kinda what confused me

Reply:
`git` ships with macOS, so, no dependency required to use it.

Reply:
done